### DISCUSSION TITLE
Error loading parsing sxcema

### DISCUSSION THREAD

**Discussion by Aggeloslou** [ckan/ckan#7704] [Q&A]
{
  "responseHeader":{
    "status":0,
    "QTime":1},
  "initFailures":{"ckan":"org.apache.solr.common.SolrException:org.apache.solr.common.SolrException: Could not load conf for core ckan: Error loading parsing schema"},
  "status":{
    "ckan":{}}}

**Comment by kowh-ai** [comment]
Are you able to log into the solr server `http://localhost:8983/solr` and see the ckan core (click on "**Core Admin**" on the left navigation)?
